compiler/rustc_errors/src/emitter.rs

```diff
@@ -2674,6 +2674,14 @@ fn from_stderr(color: ColorConfig) -> Destination {
     }
 }
 
+/// On Windows, BRIGHT_BLUE is hard to read on black. Use cyan instead.
+///
+/// See #36178.
+#[cfg(windows)]
+const BRIGHT_BLUE: Color = Color::Cyan;
+#[cfg(not(windows))]
+const BRIGHT_BLUE: Color = Color::Blue;
```

Comment:
Can you actually turn this into a single constant with `if cfg!` inside? I think it would read better ("a constant with  target-dependent value" vs "two constants with the same name being chosen per-target")